src/main/kotlin/io/snyk/plugin/SnykBulkFileListener.kt

```diff
@@ -146,19 +184,31 @@ class SnykBulkFileListener : BulkFileListener {
             // clean .dcignore caches if needed
             SnykCodeIgnoreInfoHolder.instance.cleanIgnoreFileCachesIfAffected(project, virtualFilesAffected)
 
+            val virtualFilesDeletedOrMoved = getAffectedVirtualFiles(
+                events,
+                classesOfEventsToFilter = listOf(VFileDeleteEvent::class.java, VFileMoveEvent::class.java)
+            )
             // clean IaC cached results for deleted/moved files
             updateIacCache(
-                getAffectedVirtualFiles(
-                    events,
-                    classesOfEventsToFilter = listOf(VFileDeleteEvent::class.java, VFileMoveEvent::class.java)
-                ),
+                virtualFilesDeletedOrMoved,
```

Comment:
the warning above would go away, if you put line 192-195 on one line ;)

Reply:
Yes, that one.

Reply:
fixed during refactoring